blocks.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"context"
+	"notes-service/grpc/notespb"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+var _ notespb.NotesServiceServer = &notesService{}
+
+func (srv *notesService) AddBlock(ctx context.Context, in *notespb.AddBlockRequest) (*emptypb.Empty, error) {
+
+	// the Id and Index are hard codded because the last PR protorepo is not merged
+	//objID, err := primitive.ObjectIDFromHex(in.Note_id)
+	var note_id = "62795c015f122130bb469849"
+
+	objID, err := primitive.ObjectIDFromHex(note_id)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Error : during get note 1")
+	}
+
+	var note *notespb.Note
+
+	err = NotesCollection.FindOne(context.TODO(), bson.M{"_id": objID}).Decode(&note)
+
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, err.Error())
+	}
+
+	if (len(note.Blocks) - 1) >= int(in.Index) {
```

Comment:
You don't need the parenthesis around the `if` statement. It's more idiomatic that way:
```suggestion
	if len(note.Blocks) - 1 >= int(in.Index) {
```